_test/test_unique_objects_container/test_unique_objects.m

```diff
@@ -257,9 +256,11 @@ function test_add_different_types(obj)
             [voc,nuix] = voc.add(obj.mi1);
 
             assertTrue( nuix>0 );
+            clOb = set_temporary_warning('off','HERBERT:unique_objects_container:invalid_argument');
             [voc,nuix] = voc.add(obj.nul_sm1);
             [~,lw] = lastwarn;
             assertEqual(lw,'HERBERT:unique_objects_container:invalid_argument')
+            clear cl0b;
 
             assertFalse( nuix>0 );
```

Comment:
```suggestion
            assertEqual(nuix, 0);
```
Not quite the same thing, but given it's a count will it ever be negative?

Reply:
yes